src/Functions/stringToH3.cpp

```diff
@@ -84,7 +84,7 @@ class FunctionStringToH3 : public IFunction
             auto h3index = h3index_source.getWhole();
 
             // convert to std::string and get the c_str to have the delimiting \0 at the end.
-            auto h3index_str = StringRef(h3index.data, h3index.size).toString();
+            auto h3index_str = std::string(std::string_view(h3index.data(), h3index.size()));
```

Comment:
https://en.cppreference.com/w/cpp/string/basic_string/basic_string 
Does (4) help here?

Reply:
Yepp and there is also a compile problem here (`h3index_str1` has no `data()` and `size()` members). Will fix, tnx.